src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalPrice = days * 40;
+
+  if (days < 3) {
+    return totalPrice;
+  }
+
+  if (days < 7) {
+    return (totalPrice - 20);
+  }
+
+  return (days * 40 - 50);
```

Comment:
```suggestion
  return (totalPrice - 50);
```

You can also add variables for numbers 20 and 50 (e.g. named minDiscount and maxDiscount)